tests/integration/test_dss.py

```diff
@@ -130,6 +130,37 @@ def test_log_command(cleanup_after_initialize) -> None:
     assert "Starting gunicorn" in result.stderr
 
 
+def test_remove_notebook(cleanup_after_initialize) -> None:
+    """
+    Tests that `dss remove` successfully removes a notebook as expected.
+    Must be run after `dss initialize`
+    """
+    # FIXME: remove the `--kubeconfig`` option
+    # after fixing https://github.com/canonical/data-science-stack/issues/37
+    kubeconfig_file = "~/.kube/config"
+    kubeconfig = lightkube.KubeConfig.from_file(kubeconfig_file)
+    lightkube_client = lightkube.Client(kubeconfig)
+
+    result = subprocess.run(
+        [
+            DSS_NAMESPACE,
+            "remove",
+            NOTEBOOK_NAME,
+            "--kubeconfig",
+            kubeconfig_file,
+        ]
+    )
+    assert result.returncode == 0
+
+    # Check if the notebook Deployment is not found in the namespace
+    with pytest.raises(ApiError):
+        lightkube_client.get(Deployment, name=NOTEBOOK_NAME, namespace=DSS_NAMESPACE)
+
+    # Check if the notebook Service is not found in the namespace
+    with pytest.raises(ApiError):
```

Comment:
I think you want to check that the exception that gets raised has code 404, otherwise you could be hitting false positives.

Reply:
agreed